src/core_ocean/Registry.xml

```diff
@@ -321,7 +321,7 @@
 					description="Coefficient of standard GM parametrization of eddy transport (Bolus component), $\kappa$. Used when config_GM_kappa_function is set to constant."
 		/>
 		<nml_option name="config_GM_closure" type="character" default_value="N2_dependent" units="unitless"
```

Comment:
shouldn't we have `constant` as the default?

Reply:
I wasn't sure what to put here.  This was what's there.  In E3SM we set it to visbeck and when we make the e3sm PR we'll set to edenGreatbatch in E3SM.  I don't think it matters too much what we set it to here.  What do you think @mark-petersen ?

Reply:
These defaults set how the code is exercised for our testing suite. Let's use the same defaults as E3SM for that reason, so set it to edenGreatbatch.